test/TestProjects/MgmtListOnly/Generated/FakeOperations.cs

```diff
@@ -688,11 +688,11 @@ public virtual Response<ChildWithPost> CreateOrUpdateChildWithPosts(ChildWithPos
         /// <summary> Retrieves information about an fake. </summary>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <returns> A collection of <see cref="ResponseNotCalledValue" /> that may take multiple service requests to iterate over. </returns>
-        public Pageable<ResponseNotCalledValue> ListResponseNotCalledValues(CancellationToken cancellationToken = default)
+        public Pageable<ResponseNotCalledValue> ListResponseNotCalledValue(CancellationToken cancellationToken = default)
```

Comment:
Paging method should also ends with s

Reply:
Because in this case, the response is not called `value`, this method is not identified as a List function, therefore we generated this function with a singular noun.
I believe every time we hit this case, it should be a swagger issue - therefore maybe we need to remove this case in the test swagger